Documentation/github-connector.md

```diff
@@ -8,7 +8,7 @@ When a client redeems a refresh token through dex, dex will re-query GitHub to u
 
 ## Configuration
 
-Register a new application with [GitHub][github-oauth2] ensuring the callback URL is `(dex issuer)/callback`. For example if dex is listening at the non-root path `https://auth.example.com/dex` the callback would be `https://auth.example.com/dex/callback`.
+Register a new application with [GitHub][github-oauth2] ensuring the callback URL is `(dex issuer)/callback`. For example if dex is listening at the non-root path `https://auth.example.com/dex` the callback would be `https://auth.example.com/dex/callback`. Please note that a user needs to mark their email id as public in GitHub, so that the ID token that gets returned for this user contains an email id field in the ID token claims.
```

Comment:
It's kinda sounds like you're saying that ID tokens are returned by GitHub.

Also might be worth marking this as a known caveat. e.g. https://github.com/coreos/dex/blob/master/Documentation/oidc-connector.md#caveats or https://github.com/coreos/dex/blob/master/Documentation/saml-connector.md#caveats